packages/shared/src/internalApi/api.ts

```diff
@@ -79,6 +79,10 @@ export async function fetchEndpoint<T>(
     }
   }
 
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
What if I want to run the dev environment locally and use internal API?

Reply:
I didn't get it. How is the question related to the code?

Reply:
nvm, I guess you can run development environment with `NODE_ENV=staging`